CHANGELOG.md

```diff
@@ -10,7 +10,7 @@
 
 ### Major Changes
 
-- This release introduces the **AWS IoT Over-the-air Update** library(Release Candidate), **backoffAlgorithm** library, and **PKCS #11** library.
+- This release introduces the **AWS IoT Over-the-air Update** library, **backoffAlgorithm** library, and **PKCS #11** library.
```

Comment:
I dont think this release introduces these libs. We should reword.

Reply:
This section is part of the previous release in December.